### ISSUE TITLE
Document how to do a review

### ISSUE THREAD

**Issue by KatrinIhler** [opencast/opencast#5968]
Currently our docs only very briefly talk about pull request reviews, mostly in relation to our rules. I think it would be nice if we had a "How to review" section that describes the usual process, any expectations, helpful commands and so on, to encourage more people to do a review.

**Comment by mtneug** [comment]
I vaguely remember @gregorydlogan doing a webinar/conference talk about this. Maybe I'm wong?

**Comment by KatrinIhler** [comment]
Yes, and so did I (in German). It's what I've been linking for the Review Friday, but some of the information is out of date and also I think it would be nice to have this in written form as well (if maybe not so detailed).
